packages/fileeditor-extension/src/index.ts

```diff
@@ -405,10 +405,6 @@ function activate(
               return aName.localeCompare(bName);
             })
             .forEach(spec => {
-              // Avoid mode name with a curse word.
-              if (spec.name.indexOf('brainf') === 0) {
```

Comment:
Is it just missing `toLowerCase`?

My slight preference would be to keep the mode, but fix this check.